src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-form"
+      action="https://loalhost:8080/api"
+      method="post"
+      data-qa="big"
+    >
+      <label>
+        <img
+          class="search-icon search-icon-large"
+          src="./images/Search.svg"
+          alt="search icon"
+        >
```

Comment:
Set this image as bg image for this label, not as img tag in html, it's not a content image to place it here